main.js

```diff
@@ -236,5 +224,5 @@ module.exports.static = express.static;
 module.exports.services = metrics.services;
 module.exports.metrics = metrics;
 module.exports.flags = flags;
-// TODO: for backwards compatiability, but modules/apps can use next-logger directly, as of v4.0.0
-module.exports.logger = nextLogger;
+// TODO: for backwards compatiability, but modules/apps can use n-logger directly, as of v4.0.0
+module.exports.logger = nLogger;
```

Comment:
can we delete this. It's led to problems before. There's a good reason to expose flags and metrics as we only want a single instance of those running in any app, but logger can be instantiated multiple times in an app AFAIK

cc @ironsidevsquincy